src/BDayCakeCandles/BirthdayCakeCandles.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\BDayCakeCandles;
+
+use InvalidArgumentException;
+
+class BirthdayCakeCandles
+{
+    public function blow($candles)
+    {
+        $candles = explode(PHP_EOL, $candles);
+        if (sizeof($candles) !== 2) {
+            throw new InvalidArgumentException('Input should have two lines with the candles amount and height respectively');
+        }
+        list($candlesQuantity, $candles) = $candles;
+        $candles = explode(' ', $candles);
```

Comment:
Eu recomendo, apenas pelo desafio, resolver esta função também usando um laço `for()` ou `while()`.